README.md

```diff
@@ -14,7 +14,7 @@ Dazu haben wir uns folgende Fragen gestellt:
 
 ## Technische Umsetzung
 
-Nach diesen Überlegungen haben wir versucht die Polygone der Wahlkreise und Wahlbezirke der Stadt Flensburg zu bekommen, jedoch leider bis heute keine Antwort erhalten. So haben wir uns entschieden, die Geodaten der plakatierten Orte nach der Spezifikation [RFC 7946](https://geojson.org/) in eine GeoJSON-Datei einzupflegen und mittels der Open Source Biblothek [LeafletJS](https://leafletjs.com/) auf Basis der [OpenSteetMap](https://www.openstreetmap.de/) Karte darzustellen. 
+Nach diesen Überlegungen haben wir die Polygone der Wahlkreise und Wahlbezirke bei der Stadt Flensburg angefragt, jedoch leider unzuverlässige Daten erhalten. So haben wir uns entschieden, lediglich Geodaten der plakatierten Orte nach der Spezifikation [RFC 7946](https://geojson.org/) in eine GeoJSON-Datei einzupflegen und mittels der Open Source Biblothek [LeafletJS](https://leafletjs.com/) auf Basis der [OpenSteetMap](https://www.openstreetmap.de/) Karte darzustellen. 
```

Comment:
@Philzen bitte verstehe mich nicht falsch, jedoch finde ich es unpassend ohne schriftliche Nachweise, zu schreiben das dies "unzuverlässige Daten" seien. Das kann ich so nicht mergen. Könntest du es so umformulieren, dass laut der Aussage mit deinem Gesprächspartner klar hervorgeht, dass diese Information aus einem privaten Telefonat mit Dir und Herrn Hillebrand stammt? Ich möchte zu bedenken geben, dass wir Herrn Hillebrand sehr dankbar für die Geodaten sind. Flensburg nimmt Vorreiterrolle ein, vielleicht kannst Du Kontakt mit der Vermessungsabteilung des TBZ Flensburg aufnehmen und darum bitten, deine Beobachtungen auszubessern und diese uns der Allgemeinheit zur Verfügung zu stellen.. https://github.com/oklabflensburg/open-local-elections-map/tree/main/resources

Reply:
I beg to differ.

- Die Formulierung, die zurzeit Live ist, ist noch unbeweisbarer und dazu schlicht falsch, denn die Shapes wurden ja geliefert
- Es ist mit zwei Klicks im mindesten beweisbar, dass die Bezirksgrenzen im Widerspruch mit den herausgegebenen tabellarischen Tabellenzuordnungen stehen

Parallel habe ich jetzt noch einmal die schriftliche Bestätigung erbeten.

BTW – ich glaube man kann hier selbst "resolutions" oder changes vorschlagen, insofern bin ich offen für andere Formulierungen Deinerseits.

Reply:
Ich habe den fraglichen Commit jetzt gedropped, damit die anderen Fixes erst einmal gemerged werden können :)